src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,126 @@
+package duke;
+
+import java.util.Scanner;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+public class Duke {
+    private static final TasksList tasks = new TasksList();
+
+    public static void main(String[] args) {
+        String rawInput;
+        Scanner scanner = new Scanner(System.in);
+        DukeCommand dukeCommand;
+
+        try {  // Initialize ./data/duke.txt to store tasks
```

Comment:
Comments should be placed above the line, indented relative to code position. In this case, the comments seem redundant (behaviour can be derived from the code) and can be removed.